macros/spec2.ejs

```diff
@@ -40,7 +40,7 @@ var status = {
   'CSS3 Basic UI'              : 'CR',
   'CSS3 Basic UI 4'            : 'ED',
   'CSS3 Box'                   : 'WD',
-  'CSS3 Box Alignment'         : 'WD',
+  'CSS3 Box Alignment'         : 'ED',
```

Comment:
The status of the spec. is WD, see https://www.w3.org/TR/css-align-3/.

Sebastian